src/web_app_skeleton/script/helpers/function_helpers

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+# This file contains a set of functions used as helpers
+# for various tasks. Read the examples for each one for
+# more information. Feel free to put any additional helper
+# functions you may need for your app
+
+
+# Returns true if the command $1 is not found
+# example:
+#   if command_not_found "yarn"; then
+#     echo "no yarn"
+#   fi
+command_not_found() {
+  ! command -v $1 > /dev/null
```

Comment:
I think it would be more flexible if  this was just “command” and the example would be “yarn -v”. Because some commands need “—version”

Reply:
I don't think that's what that means. I'm under the impression that the `-v` flag is on `command`, and not `$1`. I'm having trouble finding the exact documentation on it. [according to this thread](https://stackoverflow.com/questions/592620/how-can-i-check-if-a-program-exists-from-a-bash-script) some say the same, and some say `-v` doesn't work in some shells (like fish). But we're only really gearing toward bash and zsh compliant.

Reply:
Oh, actually, this confirms it:

```
[14:50PM] myapp (master)$ command -v yarn
/usr/local/bin/yarn
[14:51PM] myapp (master)$ command yarn
yarn install v1.19.1
[1/4] 🔍  Resolving packages...
success Already up-to-date.
✨  Done in 0.46s.
[14:51PM] myapp (master)$ command --version yarn
zsh: command not found: --version
```

The `command -v` is basically replacing `which` which seems to be not as posix compliant? 🤔  so I'm gonna keep it like this.